src/main/java/com/mainstreethub/ecr/LoginMojo.java

```diff
@@ -0,0 +1,62 @@
+package com.mainstreethub.ecr;
+
+import com.amazonaws.AmazonServiceException;
+import com.amazonaws.services.ecr.AmazonECR;
+import com.amazonaws.services.ecr.AmazonECRClient;
+import com.amazonaws.services.ecr.model.AuthorizationData;
+import com.amazonaws.services.ecr.model.GetAuthorizationTokenRequest;
+import com.amazonaws.services.ecr.model.GetAuthorizationTokenResult;
+import com.google.common.base.Splitter;
+import com.google.common.io.BaseEncoding;
+import java.util.List;
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.project.MavenProject;
+
+@Mojo(name = "login")
+public class LoginMojo extends AbstractMojo {
+  private static final Splitter TOKEN_SPLITTER = Splitter.on(":").limit(3).trimResults();
```

Comment:
It looks like the split is after the base64 decoding not before.  Because of that I feel like this will break if the password has a colon character in it.  What am I missing?